spec/features/create_new_work_spec.rb

```diff
@@ -154,6 +154,8 @@
           click_button('Choose files')
         end
 
+        sleep 1 # pause to wait for file to upload
```

Comment:
I wanted to test this locally but I can't get the dang tests to fail: instead of sleeping for a fixed, hard-coded value, I wonder if we can get Capybara to do this for us via:

```suggestion
        expect(page).to have_content('sul.svg')
```

Happy to pair too since apparently you *can* see this fail locally.

Reply:
This didn't do the trick.